app/src/interfaces/collection-item-dropdown/collection-item-dropdown.vue

```diff
@@ -139,6 +138,21 @@ function onSelection(selectedIds: (number | string)[] | null) {
 <style lang="scss" scoped>
 @use '@/styles/mixins';
 
+.v-list-item {
+	&:focus-within,
+	&:focus-visible {
+		--v-list-item-border-color: var(--v-input-border-color-focus, var(--theme--form--field--input--border-color-focus));
+		--v-list-item-border-color-hover: var(--v-list-item-border-color);
+
+		offset: 0;
```

Comment:
Is this supposed to be [offset](https://developer.mozilla.org/en-US/docs/Web/CSS/offset)?